### PR TITLE
feat(Tooltip): Add docs, add more complex examples (#1034)

### PR DESCRIPTION
**Pull request by essj**
Resolves [#1034](https://github.com/binance-chain/ui-project-management/issues/1034).

image

### CODE REVIEW

File: src/components/Tooltip/component.stories.tsx [hunk 1]
```diff
@@ -10,13 +10,17 @@ import { Button } from '../Button';
...
+import { docs } from './docs';
 import { RADII, SIZES, VARIANTS } from './styled';
 
 import { Tooltip } from './';
 
 export default {
   component: Tooltip,
   title: `${Sections.Elements}/Tooltip`,
+  parameters: {
+    docs: docs,
```
Comment (cchungmin) [inline src/components/Tooltip/component.stories.tsx]: Let's also use a shorthand here?

**Review by cchungmin — COMMENTED** [review]
Great! Just a comment

File: src/components/Tooltip/component.stories.tsx (hunk 1 above)
Comment (essj) [reply src/components/Tooltip/component.stories.tsx]: Done!

**Review by cchungmin — APPROVED** [review]
LGTM 👍

**Pull request merged**